### PR TITLE
feat: add consumption from specific offset

### PR DESCRIPTION
**Pull request by lvermeulen**

### CODE REVIEW

**Comment by lvermeulen** [comment]
please squash

File: src/Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple/KafkaConsumer.cs [hunk 1]
```diff
@@ -26,37 +28,47 @@ public static class KafkaConsumer
...
                 consumer.Subscribe(topic);
 
+                var kafkaJsonMessage = new KafkaJsonMessage("", "");
                 while (!cancellationToken.IsCancellationRequested)
                 {
                     var consumeResult = consumer.Consume(TimeSpan.FromSeconds(3));
                     if (consumeResult == null) //if no message is found, returns null
                     {
-                        continue;
+                        break;
```
Comment (ArneD) [inline src/Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple/KafkaConsumer.cs:53]: What's the reason for this change?
With `continue` it kept retrying, because with first time connection it could timeout.

**Review by ArneD — COMMENTED** [review]
While squashing we should add a breaking change message.
Contract of the consumer has changed.

File: src/Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple/KafkaConsumer.cs (hunk 1 above)
Comment (ArneD) [reply src/Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple/KafkaConsumer.cs:53]: There was also another reason: when the consumer reached offset 0 and polls for next message it will return `null`.
So with this change, once the consumer is up to date we stop listening.
